lib/ts/recipe/emailpassword/types.ts

```diff
@@ -272,6 +282,8 @@ export type SignInAndUpProps = BaseProps & {
 
     onCallSignUpAPI?: (requestJson: RequestJson, headers: HeadersInit) => Promise<SignUpAPIResponse>;
 
+    verifyEmailExists?: (value: string, headers: HeadersInit) => Promise<VerifyEmailAPIResponse>;
```

Comment:
We should change the name to be consistent with others: `onCallEmailExistsAPI(email: string, header: ...)`